src/models/timelines.rs

```diff
@@ -0,0 +1,130 @@
+    pub project_card: Option<ProjectCard>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub project_id: Option<ProjectId>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub project_url: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub column_name: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub assignees: Option<Vec<Author>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub assigner: Option<Author>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub updated_at: Option<chrono::DateTime<chrono::Utc>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub author_association: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub body: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub user: Option<Author>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub html_url: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub issue_url: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub tree: Option<repos::CommitObject>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub verification: Option<repos::Verification>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub parents: Option<Vec<repos::Commit>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub message: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub committer: Option<CommitAuthor>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub author: Option<CommitAuthor>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub sha: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub source: Option<Source>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub milestone: Option<Milestone>, // differs from other milestones the API returns. Has only a title.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub label: Option<Label>, // differs from other labels the API returns. Has only a name and a color.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub lock_reason: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub previous_column_name: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub rename: Option<Rename>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub submitted_at: Option<chrono::DateTime<chrono::Utc>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub state: Option<pulls::ReviewState>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub dismissed_review: Option<DismissedReview>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub pull_request_url: Option<Url>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub requested_reviewers: Option<Vec<Author>>,
```

Comment:
This should have been `pub requested_reviewers: Option<Author>`. It's only a single reviewer that's requested per event and not multiple.

Reply:
fixed with https://github.com/XAMPPRocky/octocrab/pull/390